concepts/05 UI Components/Autocomplete/00 Getting Started with Autocomplete/20 Customize Field Appearance.md

```diff
@@ -0,0 +1,89 @@
+Specify the [height](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#height) and [width](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#width) properties to change the size of the Autocomplete text field. If you want to resize the drop-down list, use the [dropDownOptions](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#dropDownOptions) object.
+
+To specify one of the predefined styles for Autocomplete text field, use the [stylingMode](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#stylingMode) property.
+
+Use the [placeholder](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#placeholder) property to give users a hint about what they should type in the Autocomplete. You can also use the [label](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#label) property for this purpose. If you specify the [label](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#label) property, set the [labelMode](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#labelMode) property to one of the following values:
+
+- *"static"*    
+The component displays the label above the input field.
+
+- *"floating"*    
+The component uses the label as a placeholder, but when the editor gets focus, the label moves to the position above the input field.
+
+- *"hidden"*    
+The label is hidden.
+
+In this tutorial, the component also uses [label](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#label) as a [placeholder](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#placeholder), because the [labelMode](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#labelMode) property is set to *"floating"*.
```

Comment:
```suggestion
In this tutorial, the component also uses the [label](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#label) as a [placeholder](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#placeholder), because the [labelMode](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#labelMode) property is set to *"floating"*.
```